README.md

```diff
@@ -3,8 +3,6 @@
 This is the Android client for [Delta Chat](https://delta.chat/).
 It is available on [F-Droid](https://f-droid.org/app/com.b44t.messenger) and
 the [Google Play Store](https://play.google.com/store/apps/details?id=chat.delta).
-The APK can also be downloaded from [GitHub](https://github.com/deltachat/deltachat-android/releases)
```

Comment:
or say `The APK can also be downloaded from [get.delta.chat](https://get.delta.chat)`